de_workloads/ingest/Ingest_AzureSql_Example/data_factory/adf_pipelines.tf

```diff
@@ -0,0 +1,35 @@
+resource "azurerm_data_factory_pipeline" "pipeline_Ingest_AzureSql_Example" {
+  count           = var.include_data_quality == false ? 1 : 0
+  name            = "Ingest_AzureSql_Example"
+  data_factory_id = data.azurerm_data_factory.factory.id
+  activities_json = file("${path.module}/pipelines/Ingest_AzureSql_Example.json")
+  description     = "Ingest from demo Azure SQL database using ingest config file."
```

Comment:
Will keep this in so the description specifically refers to the name of data source (when we template this, it can update based on the data source name).